gatling-jms/src/main/scala/io/gatling/jms/jndi/JmsJndiConnectionFactoryBuilder.scala

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2011-2017 GatlingCorp (http://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gatling.jms.jndi
+
+import java.util.{ Hashtable => JHashtable }
+import javax.jms.ConnectionFactory
+import javax.naming.{ Context, InitialContext }
+
+import com.typesafe.scalalogging.StrictLogging
+import io.gatling.core.config.Credentials
```

Comment:
Our import sorting rule is:
* first java.*
* then scala.*
* then io.gatling.*
* then others